ipfsconn/ipfshttp/config.go

```diff
@@ -69,6 +81,9 @@ type jsonConfig struct {
 	ProxyWriteTimeout       string `json:"proxy_write_timeout"`
 	ProxyIdleTimeout        string `json:"proxy_idle_timeout"`
 	PinMethod               string `json:"pin_method"`
+	ClientPostTimeout       string `json:"client_post_timeout"`
```

Comment:
`ClientPost` seems a too internal name for a user. Better `IPFSTimeout` or `IPFSRequestTimeout`

Reply:
✅